governance/votes/Executive vote - June 13, 2024.md

```diff
@@ -0,0 +1,169 @@
+		
+- **Authorization:** [Atlas 2.5.10 prior to its removal by MIP102c2-SP34](https://github.com/makerdao/mips/pull/1100/files#diff-bad206bdb2371d68c4940c519b4cf8817b59781422c6d7c0d8da48ab24decb33L535-L565)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/avc-member-participation-rewards-q2-2024/24442)
+		
+If this executive proposal passes, the following distributions totaling **100.08 MKR** will be made for Q2 2024 AVC member compensation. This has been pro-rated to two months following the most recent amendments to The Atlas that removed the AVC Member role.
+		
+| AVC Member Name | Address                                                                                                               | AVC                | Amount (MKR) | 
+|-----------------|-----------------------------------------------------------------------------------------------------------------------|--------------------|--------------|
+| IamMeeoh        | [0x47f7A5d8D27f259582097E1eE59a07a816982AE9](https://etherscan.io/address/0x47f7A5d8D27f259582097E1eE59a07a816982AE9) | KISS               | 12.51        |       
+| DAI-Vinci       | [0x9ee47F0f82F1A6F45C4E1D25Ce95C321D8C8356a](https://etherscan.io/address/0x9ee47F0f82F1A6F45C4E1D25Ce95C321D8C8356a) | KISS               | 12.51        |       
+| opensky         | [0xf44f97f4113759E0a57756bE49C0655d490Cf19F](https://etherscan.io/address/0xf44f97f4113759E0a57756bE49C0655d490Cf19F) | KISS               | 12.51        |       
+| Res             | [0x8c5c8d76372954922400e4654AF7694e158AB784](https://etherscan.io/address/0x8c5c8d76372954922400e4654AF7694e158AB784) | Resiliency         | 12.51        |       
+| Harmony         | [0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3](https://etherscan.io/address/0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3) | Resiliency         | 12.51        |       
+| Libertas        | [0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f](https://etherscan.io/address/0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f) | Resiliency         | 12.51        |       
+| seedlatam.eth   | [0xd43b89621fFd48A8A51704f85fd0C87CbC0EB299](https://etherscan.io/address/0xd43b89621fFd48A8A51704f85fd0C87CbC0EB299) | Sovereign Finance  | 12.51        |       
+| 0xRoot          | [0xC74392777443a11Dc26Ce8A3D934370514F38A91](https://etherscan.io/address/0xC74392777443a11Dc26Ce8A3D934370514F38A91) | Sovereign Finance  | 12.51        |
+		
+### Launch Project Funding
+		
+- **Authorization:** [Accessibility Scope 9.1](https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/17)
+		
+If this executive proposal passes, the following transfers to the Launch Project will be made:
+		
+- Transfer **5,000,000 DAI** to the Launch Project at [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
```

Comment:
I'd change the verb to the noun (by adding "of" after "Transfer") to better fit the flow of the sentence. This comment applies also to the second bullet.

Reply:
This is the standard language that has been used on executives and polls. We can consider changing in the style guide, but I do not think it is necessary here.